cmd/boulder-ca/main.go

```diff
@@ -203,8 +218,34 @@ func main() {
 		issuers = append(issuers, issuer)
 	}
 
-	profile, err := issuance.NewProfile(c.CA.Issuance.Profile, c.CA.Issuance.IgnoredLints)
-	cmd.FailOnError(err, "Couldn't load issuance profile")
+	if c.CA.Issuance.DefaultCertificateProfileName == "" {
+		c.CA.Issuance.DefaultCertificateProfileName = "defaultBoulderCertificateProfile"
+	}
+	logger.Infof("Configured default certificate profile name set to: %s", c.CA.Issuance.DefaultCertificateProfileName)
+
+	deprecatedProfileExists := reflect.ValueOf(c.CA.Issuance.Profile).IsZero()
+	if !deprecatedProfileExists && len(c.CA.Issuance.CertProfiles) > 0 {
```

Comment:
The variable name here is backwards -- this is true when the deprecated profile _doesn't_ exist. But even better, I think this can be simplified to
```suggestion
	if (c.CA.Issuance.Profile != issuance.ProfileConfig{}) && len(c.CA.Issuance.CertProfiles) > 0 {
```

Reply:
That throws error
```
invalid operation: c.CA.Issuance.Profile != issuance.ProfileConfig{} (struct containing []issuance.PolicyConfig cannot be compared)compiler[UndefinedOp](https://pkg.go.dev/golang.org/x/tools/internal/typesinternal#UndefinedOp)
```

I'll rename that variable to `deprecatedProfileAbsent`.

Reply:
Or you could keep the name and make it `deprecatedProfileExists := !reflect...`.

Reply:
I did something similar a few minutes ago. 👍🏼